core/src/main/java/org/apache/accumulo/fate/zookeeper/ZooReaderWriter.java

```diff
@@ -40,7 +40,7 @@ public interface Mutator {
     byte[] mutate(byte[] currentValue) throws AcceptableThriftTableOperationException;
   }
 
-  public ZooReaderWriter(SiteConfiguration conf) {
+  public ZooReaderWriter(AccumuloConfiguration conf) {
```

Comment:
Why is this change necessary?  I believe site configuration has the ZK info, I'm not sure that an AccumuloConfiguration must have it. It could be a subset. Or, maybe I have it backward?

Reply:
You may be correct - it looks like https://github.com/apache/accumulo/pull/2215 wanted to get away from SiteConfiguration